### ISSUE TITLE
animateToPage in iOS

### ISSUE THREAD

**Issue by jesus-abraham** [serenader2014/flutter_carousel_slider#82]
I have a carousel with 3 elements. Each element has a button to go to next element. Button of slide index 0 works well, but button of slide index 1 jumps to slide index 3 instead of slide index 2. That issue only happens in iOS.

I've made a hard fix that works well but I think it wouldn't work like this:

```
slider.animateToPage(
    Platform.isIOS && position == 1 ? 1 : (position + 1),
    duration: Duration(milliseconds: 300),
    curve: Curves.ease,
);
```

**Comment by bubbleis** [comment]
the same issue on Android, using .jumpToPage or AnimateToPage causes the same result instead of animating to right position it is jumping to given position + 2  :(

@serenader2014

**Comment by jesus-abraham** [comment]
> the same issue on Android, using .jumpToPage or AnimateToPage causes the same result instead of animating to right position it is jumping to given position + 2 :(
> 
> @serenader2014

@bubbleis my solution is that always use
```
slider.animateToPage(
    1,
    duration: Duration(milliseconds: 300),
    curve: Curves.ease,
);
```

It works for me.

**Comment by serenader2014** [comment]
Hi @jesus-abraham , sorry for the late response. I think this issue has been fixed in the v2.0.0 version, please upgrade the carousel_slider to v2.0.0.

Let me know if the issue still exist.
